### PR TITLE
Add views to database: vUserProgress & vUserTasks

### PR DESCRIPTION
**Pull request by STALINovsky**
I've got small problem with naming: word Progress does'nt has a plural form.It has only, when we talk about many types of progress, it's like word - fish.

### CODE REVIEW

**Review by dbogdan2000 — APPROVED** [review]
Everything is fine

File: DIMS-Core.Database/DIMS-Core.Database/Views/vUserProgress.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        UserProfiles.UserId,
+        UserTasks.TaskId,
+        TaskTracks.TaskTrackId,
+        (UserProfiles.FirstName + ' ' + UserProfiles.LastName) as UserName,
+        Tasks.[Name] as TaskName,
+        TaskTracks.TrackNote,
+        TaskTracks.TrackDate
+        FROM UserProfiles
+        INNER JOIN UserTasks on UserProfiles.UserId = UserTasks.UserId
+        INNER JOIN TaskTracks on UserTasks.TaskId = TaskTracks.UserTaskId
```
Comment (PushMeNow) [inline DIMS-Core.Database/DIMS-Core.Database/Views/vUserProgress.sql:12]: You cannot join UserTasks and TaskTracks table by TaskId -> UserTaskId. Incorrect matching.

File: DIMS-Core.Database/DIMS-Core.Database/Views/vUserTracks.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿CREATE VIEW [dbo].[vUserTracks]
+	AS SELECT 
+		UserTasks.UserId,
+		Tasks.TaskId,
+		TaskTracks.TaskTrackId,
+		Tasks.[Name] as TaskName,
+		TaskTracks.TrackNote,
+		TaskTracks.TrackDate
+		FROM UserTasks
+		INNER JOIN TaskTracks on UserTasks.TaskId = TaskTracks.UserTaskId
```
Comment (PushMeNow) [inline DIMS-Core.Database/DIMS-Core.Database/Views/vUserTracks.sql:10]: And again. You cannot join UserTasks and TaskTracks table by TaskId -> UserTaskId. Incorrect matching.

**Review by PushMeNow — CHANGES_REQUESTED** [review]
Hi.
I did some comments. Let me know if any questions.

**Comment by STALINovsky** [comment]
@PushMeNow i fixed it.

**Review by maximsan — APPROVED** [review]
Looks ok

**Review by PushMeNow — APPROVED** [review]
Approved

**Pull request merged**